tests/performance/arrow_format.xml

```diff
@@ -0,0 +1,14 @@
+<test>
+    <create_query>CREATE TABLE test (uint32 UInt32, n_uint32 Nullable(UInt32), lc LowCardinality(String)) ENGINE=File(Arrow) SETTINGS output_format_arrow_low_cardinality_as_dictionary=1</create_query>
+    <create_query>CREATE TABLE test_lc_nullable (lc LowCardinality(Nullable(String))) ENGINE=File(Arrow) SETTINGS output_format_arrow_low_cardinality_as_dictionary=1</create_query>
```

Comment:
Just wondering if there is a difference between `LowCardinality(Nullable(String))` and `Nullable(LowCardinality(String))`? Asking because above in the source code you are using method `DataTypeLowCardinality::isLowCardinalityNullable()` and maybe there is something like `DataTypeNull::isNullableLowCardinality()`.

Reply:
There is no type `Nullable(LowCardinality(String))` in ClickHouse, we don't allow `LowCardinality` inside `Nullable`

Reply:
Good.